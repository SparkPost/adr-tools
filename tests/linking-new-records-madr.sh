adr init docs madr
adr new First Record
adr new Second Record
adr new -l "2:Amends:Amended by" -l "3:Clarifies:Clarified by" Third Record
head -12 docs/0002-first-record.md
head -12 docs/0003-second-record.md
head -14 docs/0004-third-record.md
