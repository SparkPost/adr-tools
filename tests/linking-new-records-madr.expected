adr init docs madr
docs/0001-record-architecture-decisions.md
adr new First Record
docs/0002-first-record.md
adr new Second Record
docs/0003-second-record.md
adr new -l "2:Amends:Amended by" -l "3:Clarifies:Clarified by" Third Record
docs/0004-third-record.md
head -12 docs/0002-first-record.md
# First Record

| Date | Status |
| -- | -- |
| 1992-01-12 | Accepted |
| 1992-01-12 | Amended by [Third Record](0004-third-record.md) |

User Story: [ticket/issue-number] <!-- optional -->

[context and problem statement]
[decision drivers | forces | facing] <!-- optional -->

head -12 docs/0003-second-record.md
# Second Record

| Date | Status |
| -- | -- |
| 1992-01-12 | Accepted |
| 1992-01-12 | Clarified by [Third Record](0004-third-record.md) |

User Story: [ticket/issue-number] <!-- optional -->

[context and problem statement]
[decision drivers | forces | facing] <!-- optional -->

head -14 docs/0004-third-record.md
# Third Record

| Date | Status |
| -- | -- |
| 1992-01-12 | Accepted |
| 1992-01-12 | Amends [First Record](0002-first-record.md) |
| 1992-01-12 | Clarifies [Second Record](0003-second-record.md) |

User Story: [ticket/issue-number] <!-- optional -->

[context and problem statement]
[decision drivers | forces | facing] <!-- optional -->

## Considered Options
